R/package.R

```diff
@@ -66,6 +66,7 @@ ensure_python_initialized <- function(required_module = NULL) {
 
   # remap output streams to R output handlers
   remap_output_streams()
+  set_knitr_python_stdout_hook()
```

Comment:
Should this hook be confined to `eng_python()`? {knitr} is only in Suggests, and to register this hook globally, we would need to move knitr to Depends.

Reply:
`eng_python` is only used when there are Python chunks in the document, but in theory this problem could hapen even in R only chunks that call reticulate. Thus we need a global knitr hook.